packages/website/app/routes/home.tsx

```diff
@@ -0,0 +1,35 @@
+import { Link, Outlet } from "@remix-run/react";
+import type { LinksFunction,MetaFunction } from '@remix-run/node' 
```

Comment:
You have removed `@remix-run/node` from `package.json`, but not here.

Reply:
ojbk